public/elements/zero-tournament.html

```diff
@@ -169,96 +178,106 @@
-                <div class="match-info" on-click="_toggleMatch">
-                  <div class="match-title">
-                    <paper-icon-button class="match-toggle"
-                        icon="hardware:keyboard-arrow-down"
-                        title="More Info"
-                        >
-                    </paper-icon-button>
-                    <div class="team-name">
-                      <div class$="score-badge {{getScoreBadgeClass(match.alpha.score)}}">{{match.alpha.score}}</div>
-                      <span><a href="#{{match.alpha.id}}" on-click="_stopPropagation">{{match.alpha.name}}</a></span>
-                    </div>
-                    vs.
-                    <div class="team-name">
-                      <span><a href="#{{match.bravo.id}}" on-click="_stopPropagation">{{match.bravo.name}}</a></span>
-                      <div class$="score-badge {{getScoreBadgeClass(match.bravo.score)}}">{{match.bravo.score}}</div>
-                    </div>
-                  </div>
-                  <iron-collapse class="match-details-collapse">
-                    <div class="match-details">
-                      <template is="dom-repeat" items="{{match.games}}" as="game">
-                        <div class="match-detail">
-                          <div class="game-title">
-                            <a href="http://destinytracker.com/dg/{{game.id}}" target="_blank">Game {{game.number}}</a>
-                          </div>
-                          <div>{{game.playlist}}</div>
-                          <div>{{game.map}}</div>
-                          <div class="game-modifier">
-                            <div>{{game.modifier.name}}</div>
-                            <paper-tooltip>{{game.modifier.description}}</paper-tooltip>
-                          </div>
-                        </div>
-                      </template>
-                    </div>
-                  </iron-collapse>
-                </div>
-              </template>
-            </div>
-          </paper-card>
-        </template>
-
-        <hr>
-        <div class="rankings">
-          <paper-card heading="Rankings" class="round-card">
-            <vaadin-grid id="rankings">
-              <table>
-                <colgroup>
-                  <col name="team" />
-                  <col name="matchesPlayed" />
-                  <col name="wins" />
-                  <col name="losses" />
-                  <col name="gamesWon" />
-                </colgroup>
-                <thead>
-                  <tr>
-                    <th>Team</th>
-                    <th>Matches played</th>
-                    <th>Wins</th>
-                    <th>Losses</th>
-                    <th>Games won</th>
-                  </tr>
-                </thead>
-              </table>
-            </vaadin-grid>
-          </paper-card>
-        </div>
+    <iron-pages id="top-pages" selected="0">
```

Comment:
This node and its first child are really the only things being added here. The rest of the DOM diffs are due to indentation changes and the introduction of new lines.